articles/changes.md

```diff
@@ -0,0 +1,241 @@
+---
+layout: default
+title: Changes between ROS 1 and ROS 2
+permalink: articles/changes.html
+abstract:
+  This article provides an overview about the changes being made in ROS 2 compared to ROS 1.
+published: true
+author: '[Dirk Thomas](https://github.com/dirk-thomas)'
+---
+
+* This will become a table of contents (this text will be scraped).
+{:toc}
+
+# {{ page.title }}
+
+<div class="abstract" markdown="1">
+{{ page.abstract }}
+</div>
+
+Original Author: {{ page.author }}
+
+
+## Preface
+
+Each change is described as briefly as possible but giving enough context and rational for a reader familiar with ROS 1.
+
+
+## Platforms and dependencies
+
+### Platforms
+
+ROS 1 is only being CI tested on Ubuntu.
+It is actively supported by the community on other Linux flavors as well as OS X.
+
+ROS 2 will be CI tested and supported on Ubuntu, OS X as well as Windows.
+
+
+### Languages
+
+#### C++ standard
+
+ROS 1 is targeting C++03.
+ROS 2 will use C++11.
+
+
+#### Python
+
+ROS 1 is targeting Python 2.
+ROS 2 will use 3.4+.
+
+
+### Reusing existing middleware
+
+ROS 1 uses a custom serialization format, a custom transport protocol as well as a custom central discovery mechanism.
+ROS 2 will use an abstract interface and provide multiple implementations of that interface based using existing DDS implementations.
+
+DDS will enable ROS 2 to provide various Quality of Service parameters which will improve communication over different networks.
+
+
+## Build system
+
+### Support other build systems beside CMake
+
+Every ROS package is a CMake project.
+In ROS 2 other build systems will be supported, e.g. plain Python packages.
+
+
+### Python packages
+
+In ROS 1 a package with Python code can only use a small subset of the features available in setup.py files.
+In ROS 2 a Python package can use anything in setup.py files, e.g. entry points.
+
+
+### Environment setup
+
+In ROS 1 the build tool generates scripts which can be sourced to setup the environment before being able to use the built ROS packages.
```

Comment:
`... which can be sourced ...` makes it sound optional and then later in the sentence it is described as a prerequisite for using the built packages. Maybe this would be more consistent: `In ROS 1 the build tool generates scripts which must be sourced in order to setup the environment before being able to use the built ROS packages.